util/src/main/java/io/kubernetes/client/Exec.java

```diff
@@ -74,12 +76,11 @@ public void setApiClient(ApiClient apiClient) {
   }
 
   private String makePath(
-      String namespace,
-      String name,
-      String[] command,
-      String container,
-      boolean stdin,
-      boolean tty) {
+      String namespace, String name, String[] command, String container, boolean stdin, boolean tty)
+      throws UnsupportedEncodingException {
```

Comment:
Hrm, I feel like populating this forward is probably not right. The encoder is _extremely_ unlikely to throw this since UTF-8 is supported everywhere (and it better be, or HTTP won't work right...)

So I would just try/catch and then throw RuntimeException or some other sort of exception that doesn't have to be caught.